schemas/jobWithLegacyBursarParameters.json

```diff
@@ -0,0 +1,101 @@
+{
```

Comment:
why do we need this schema?

Reply:
So this one can be autogenerated, too. @danetsao please confirm we do need this for mdes.

Reply:
Yes it is needed. Here is the [schema](https://github.com/folio-org/mod-data-export-spring/pull/270/files#diff-cbf72f961becc6965ae285c6984b419f2a60684eb6a96665f3ae18bd8273a10f) and [file](https://github.com/folio-org/mod-data-export-spring/pull/270/files#diff-cbf72f961becc6965ae285c6984b419f2a60684eb6a96665f3ae18bd8273a10f:~:text=src/main/java/org/folio/de/entity/bursarlegacy/JobWithLegacyBursarParameters.java), and it's usage in [BursarMigrationService](https://github.com/folio-org/mod-data-export-spring/pull/270/files#diff-f545f1994f6119b66c9d8fb6cd9dac7565c5cc26651240d16dadd58f742f73f5).

Reply:
I'm not sure you need this migration, let's keep this thread open before we understand that this migration is correct and required